woocommerce/woocommerce
# Update block templates documentation

### Changes proposed in this Pull Request:

This PR updates the internal documentation we have related to block templates, which has been out of date for some time, specially since https://github.com/woocommerce/woocommerce/pull/44537.

Note: this PR is not expanding or improving the documentation in any way, as this part of the codebase is likely to change in the future (https://github.com/WordPress/gutenberg/issues/41401), but, at least, I wanted to make sure there was no incorrect info. :slightly_smiling_face: 

Closes https://github.com/woocommerce/woocommerce/issues/47043.

### How to test the changes in this Pull Request:

_(No need to test during the release)_

1. Read the docs.
2. Verify they are understandable and make sense.

### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
